lib/methods/index.js

```diff
@@ -178,6 +178,7 @@ module.exports = (
     state = state || uuidv4()
     redirectUri = redirectUri || config.redirectUriFqdn
     serviceId = serviceId || config.serviceId
+    clientId = clientId || config.clientId
```

Comment:
should this be options.clientId and options.serviceId on the line above?

Reply:
That `options` is the name of the third input parameter object - so no